internal/provider/resource_port_loa_test.go

```diff
@@ -0,0 +1,30 @@
+//go:build resource || core || all
+
+package provider
+
+import (
+	"testing"
+
+	"github.com/PacketFabric/terraform-provider-packetfabric/internal/testutil"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccPortLOARequiredFields(t *testing.T) {
+	portLoaResult := testutil.RHclPortLoa()
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
```

Comment:
the PreCheck section can be removed as it must be set before the test to check env variable